### PR TITLE
Adding Changelog and Acknowledgements

### PR DESCRIPTION
**Pull request by wareid**
Adding changelog and acknowledgements sections.

If I have forgotten anyone, please let me know!

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/PWETF/pull/303.html" title="Last updated on Jul 13, 2023, 3:10 PM UTC (0bcfdfc)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/PWETF/303/b58dbfb...0bcfdfc.html" title="Last updated on Jul 13, 2023, 3:10 PM UTC (0bcfdfc)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -868,5 +868,76 @@ <h2>
...
+          16-May-2023: Edited the <a>Abstract</a> to better reflect the goals and content of the CoC. See <a href="https://github.com/w3c/PWETF/pull/292">PR #292</a>.
+        </li>
+        <li>
+          16-May-2023: Changed the unordered lists to numbered ones. See <a href="https://github.com/w3c/PWETF/pull/291">PR #291</a> and <a href="https://github.com/w3c/PWETF/issues/259">issue #259</a>.
+        </li>
+        <li>
+          16-May-2023: Revise the section on safety vs comfort for clarity. See <a href="https://github.com/w3c/PWETF/pull/290">PR #290</a> and <a href="https://github.com/w3c/PWETF/pull/251">PR #251</a>.
+        </li>
+        <li>
+          16-May-2023: Added editorial recommendations from three issues (<a href="https://github.com/w3c/PWETF/issues/260">#260</a>, <a href="https://github.com/w3c/PWETF/issues/261">#261</a>, and <a href="https://github.com/w3c/PWETF/issues/262">#262</a>), regarding wording and structure for some points in the behaviour sections. See <a href="https://github.com/w3c/PWETF/pull/287">PR #287</a>.
```
Comment (TzviyaSiegman) [inline index.html:897]: behavior (american spelling)

File: index.html [hunk 2]
```diff
@@ -868,5 +868,76 @@ <h2>
...
+          27-Sep-2022: Changed the language for the behaviour on threats from "physical threats". See <a href="https://github.com/w3c/PWETF/pull/206">PR #206</a>.
+        </li>
+        <li>
+          27-Sep-2022: Add "personal attacks" as an unacceptable behaviour. See <a href="https://github.com/w3c/PWETF/pull/184">PR #184</a>.
+        </li>
+        <li>
+          30-Aug-2022: Change the word "member" to "participant". See <a href="https://github.com/w3c/PWETF/pull/208">PR #208</a>.
+        </li>
+        <li>
+          7-Jun-2022: Added "deliberate misinformation" as an unacceptable behaviour. See <a href="https://github.com/w3c/PWETF/pull/201">PR #201</a>.
```
Comment (TzviyaSiegman) [inline index.html:927]: behavior (US spelling)

**Review by TzviyaSiegman — CHANGES_REQUESTED** [review]

File: index.html [hunk 3]
```diff
@@ -937,7 +937,32 @@ <h2>Change Log</h2>
     <section id="acknowledgements">
       <h2>Acknowledgements</h2>
 
+      <p>We'd like to acknowledge the efforts of the editors of the 2020 revision of the CEPC, Ada Rose Cannon, An Qi Li, and Tzviya Siegman, whose work got us to the revision we use today. Editing a code of conduct is a challenging task, and their insight and leadership is evident in the quality of the document we continue to work on today.</p>
 
+      <p>The following people contributed to the development of this code:</p>
+
+      <ul class="ack">
+        <li>David Booth (W3C Invited Expert)</li>
```
Comment (TzviyaSiegman) [inline index.html:945]: Judy Brewer

Comment (TzviyaSiegman) [reply index.html:945]: Jory Burson

File: index.html [hunk 4]
```diff
@@ -937,7 +937,32 @@ <h2>Change Log</h2>
...
+        <li>Nick Doty (Center for Democracy and Technology)</li>
+        <li>Annette Greiner (Lawrence Berkeley National Laboratory)</li>
+        <li>Amy van der Hiel (W3C)</li>
+        <li>Barbara Hochgesang (Intel Corporation)</li>
+        <li>JaEun Jemma Ku (University of Illinois)</li>
```
Comment (TzviyaSiegman) [inline index.html:950]: Vlad Levantovsky

File: index.html (hunk 3 above)
Comment (TzviyaSiegman) [inline index.html:945]: first thanks is to Ann Bassetti

File: index.html [hunk 5]
```diff
@@ -937,7 +937,32 @@ <h2>Change Log</h2>
...
+        <li>Liz Lutgendorff (HM Government)</li>
+        <li>Nigel Megitt (BBC)</li>
```
Comment (TzviyaSiegman) [inline index.html:952]: Coralie Mercier, W3C

File: index.html (hunk 3 above)
Comment (TzviyaSiegman) [reply index.html:945]: Dan Appelquist

File: index.html [hunk 6]
```diff
@@ -937,7 +937,32 @@ <h2>Change Log</h2>
...
+        <li>Sheila Moussavi (Bocoup)</li>
+        <li>Charles Nevile (ConsenSys)</li>
+        <li>Theresa O'Connor (Apple Inc.)</li>
+        <li>Wendy Reid (Rakuten Group, Inc.)</li>
+        <li>Cybele Sack (W3C Invited Expert)</li>
+        <li>Ralph Swick (W3C)</li>
+        <li>Tzviya Siegman (Wiley)</li>
+        <li>Jeanne F Spellman (Tetralogical)</li>
+        <li>Jennifer Stricßkland (MITRE Corporation)</li>
```
Comment (TzviyaSiegman) [inline index.html:961]: extra character in Strickland

File: index.html [hunk 7]
```diff
@@ -937,7 +937,32 @@ <h2>Change Log</h2>
...
+        <li>Barbara Hochgesang (Intel Corporation)</li>
+        <li>JaEun Jemma Ku (University of Illinois)</li>
+        <li>Liz Lutgendorff (HM Government)</li>
+        <li>Nigel Megitt (BBC)</li>
+        <li>Sheila Moussavi (Bocoup)</li>
+        <li>Charles Nevile (ConsenSys)</li>
+        <li>Theresa O'Connor (Apple Inc.)</li>
+        <li>Wendy Reid (Rakuten Group, Inc.)</li>
+        <li>Cybele Sack (W3C Invited Expert)</li>
+        <li>Ralph Swick (W3C)</li>
```
Comment (TzviyaSiegman) [inline index.html:958]: Swick is out of alpha, Add Wendy Seltzer

**Review by TzviyaSiegman — CHANGES_REQUESTED** [review]

**Comment by wareid** [comment]
Names added and fixed the Canadian-isms.

**Pull request merged**